examples/musicdiff_demo.ipynb

```diff
@@ -0,0 +1,533 @@
+{
```

Comment:
The amount of data here that is specific to your run of this colab is distressing.  I don't know much about colab; is that extra output here supposed to be committed?